dbcon/joblist/batchprimitiveprocessor-jl.h

```diff
@@ -154,8 +151,7 @@ class BatchPrimitiveProcessorJL
     /* Turn a ByteStream into ElementTypes or StringElementTypes */
     void getElementTypes(messageqcpp::ByteStream& in, std::vector<ElementType>* out,
```

Comment:
It looks like we even don't use BatchPrimitiveProcessorJL::getElementTypes.

Reply:
True. As well as a number of other "get" functions in that class. This is beyond the scope of this work. MCOL-4045 has been opened to track this.

Reply:
Yeah all of the "ElementType" stuff is likely OBE.  It's from the very first, and low-performing version of calpont.  '08 or so.

Reply:
This is good to know. I also came across getElementType(), getStringElementTypes(), etc. at some point and wasn't sure if these were being used anywhere.